music21/romanText/tsvConverter.py

```diff
@@ -692,14 +751,17 @@ def m21ToTsv(self):
             return self._m21ToTsv_v1()
         return self._m21ToTsv_v2()
 
-    def _m21ToTsv_v1(self):
+    def _m21ToTsv_v1(self) -> t.List[t.List[str]]:
         tsvData = []
         # take the global_key from the first item
         global_key = next(
             self.m21Stream.recurse().getElementsByClass('RomanNumeral')
         ).key.tonicPitchNameWithCase
 
         for thisRN in self.m21Stream[roman.RomanNumeral]:
+            if thisRN is None:
+                # shouldn't occur, but to satisfy mypy
+                continue
```

Comment:
that is strange -- mypy should know that it will be a roman.RomanNumeral -- have you merged master from music21 recently?

Reply:
Yes, and I'm not getting that warning again after deleting these lines.